lib/seahorn/BvOpSem2ExtraWideMemMgr.cc

```diff
@@ -407,12 +407,11 @@ ExtraWideMemManager<T>::ptrAdd(ExtraWideMemManager::PtrTy base,
   // do concrete computation if possible
   if (m_ctx.alu().isNum(base.getOffset())) {
     signed conc_offset = m_ctx.alu().toNum(base.getSize()).get_si() + _offset;
```

Comment:
unrelated to this change:  `s/base.getSize()/base.getOffset()`

Reply:
fixed